### DISCUSSION TITLE
Saltwater Fish Submissions 🐟

### DISCUSSION THREAD

**Discussion by BirbSunflower** [sunflower-land/sunflower-land#3051] [Art]
With fish taking center stage in the upcoming season, as a new resource brimming with boundless design potential, the Art Team is excited to open this discussion, inviting the community to contribute their creative fish designs! 😍

For this discussion, we are looking for **only saltwater fish** and **sea creatures**!

The more diverse colors, shapes, and perhaps even representations of fish from your own region or country, the better! 🐟

You can read more about fishing [here](https://docs.sunflower-land.com/player-guides/fishing).

## Requirements ✏️

- The art should be original and not plagiarised.
- By submitting art, you acknowledge you are the original creator of the art and you give Thought Farm and the community the rights to use it freely in the game and marketing purposes indefinitely.
- You agree to the [terms and services](https://docs.sunflower-land.com/contributing/competitions/fan-art-competition#terms-and-conditions) of the game.

When submitting, do not forget: 
- Your farm ID;
- Your discord ID;
- Name of the fish;
- Rarity of the fish (common, uncommon, rare );
- A true to size version of your art piece ( maximum size: 16x16 pixels );
- A 400% zoom version or higher;
- Use the following fish design as a reference: 

In game version:  

| ![image](https://github.com/sunflower-land/sunflower-land/assets/123522693/4b404b10-66ff-48b4-9ed0-e8ef7aa8a985) |

600% zoom:

 ![image](https://github.com/sunflower-land/sunflower-land/assets/123522693/d5589b25-9282-4aa9-894b-039d48755536)

## Rewards 🏆

There is no limit to the number of submissions that can be accepted. The artist will receive 250 SFL for each design that makes it into the game.

Not only will the artwork be showcased on the official Sunflower Land Game social media channels and in-game, but it will also be seen by our vibrant community of players from around the world.

Sunflower Land is a community game and we love showcasing and including artwork from our players. Now is your chance to have your artwork immortalised in the #1 Community Web3 game.❤

## Deadline 📅

No deadline required! 😍
The Art Team will periodically review this discussion.

Good luck to all participants! 🎉

**Comment by davosalm** [comment]
First suggestion: Garibaldi

![GARIBALDIBIG](https://github.com/sunflower-land/sunflower-land/assets/9125640/3e312084-df33-44df-bb7d-0c6318493450)
![GARIBALDISMALL](https://github.com/sunflower-land/sunflower-land/assets/9125640/04643eb7-503d-42f4-9347-342d1938660f)


Farm ID: 99994
Discord ID: Caramelaikos
Rarity: Advanced

**Comment by SaqibUX** [comment]
Farm: 170828
Discord: sar_01
Fish Name: Goldy
Rarity: Basic
16x16
![Goldy16](https://github.com/sunflower-land/sunflower-land/assets/149222417/fb3b2d29-c730-4602-aae9-7e3faa0d1dfa)
256x256
![Goldy](https://github.com/sunflower-land/sunflower-land/assets/149222417/4178db6c-7782-4b4b-b8ce-ba95017af31b)

**Comment by Thebyber** [comment]
Farm: 16230
Discord: Thebyber
Fish Name: Flying fish
Rarity: Basic
16x16
![Flying fish 100](https://github.com/sunflower-land/sunflower-land/assets/124185010/8c3690c0-8dd8-41f7-967a-f33d8d24a6d3)
400%
![Flying fish](https://github.com/sunflower-land/sunflower-land/assets/124185010/e7742e34-a83d-4d7d-b6a7-3b5c025eda41)

Flying fish is typical of my city (Southern Spain)! It is usually dried and eaten! 
![image](https://github.com/sunflower-land/sunflower-land/assets/124185010/501db7cc-7edc-43e0-b3e4-f80b021c3e07)

**Comment by SaqibUX** [comment]
**Farm: 170828
Discord: sar_01**

Fish Name: Pascal
Rarity: Advanced
16x16
![pascel16](https://github.com/sunflower-land/sunflower-land/assets/149222417/0ecb6e9b-02a4-4ec1-84a1-e88fd79faefe)

128x128
![Pascal256](https://github.com/sunflower-land/sunflower-land/assets/149222417/859b456b-08cb-4e9c-ae4e-5a24ff8f97fe)


Fish Name: Horse
Rarity: Basic
16x16
![horse16](https://github.com/sunflower-land/sunflower-land/assets/149222417/db6b0691-b557-4b7f-ac58-58bcc693f0ad)
128x128
![horse](https://github.com/sunflower-land/sunflower-land/assets/149222417/6a1ce2e4-27ff-4a71-b3a6-9ead95333ff1)

Fish Name: StinkyCrown
Rarity: Basic
16x16
![spikycrown16](https://github.com/sunflower-land/sunflower-land/assets/149222417/a4450b7c-9662-4e34-b725-cd3880351d39)

128x128
![spikycrown](https://github.com/sunflower-land/sunflower-land/assets/149222417/4ee0b93d-e119-432a-be80-4c220212a3e1)


Fish Name: Spikestrips
Rarity: Advanced
16x16
![spikestripe16](https://github.com/sunflower-land/sunflower-land/assets/149222417/6dbe9388-6838-44c1-95d9-015024889942)

128x128
![spikestripe](https://github.com/sunflower-land/sunflower-land/assets/149222417/f9135b1d-d8c4-4113-99a5-6c3d12c4f8a2)


Fish Name: Bunnyfish
Rarity: Basic
16x16
![bunnyfish16](https://github.com/sunflower-land/sunflower-land/assets/149222417/a7945f95-d9ee-4a2f-a2b7-6bc54bbbce1e)

128x128
![bunnyfish](https://github.com/sunflower-land/sunflower-land/assets/149222417/c7132302-fec3-45e7-bc71-2d80b5fe5765)


Fish Name: Speedster
Rarity: Expert
16x18
![horn16](https://github.com/sunflower-land/sunflower-land/assets/149222417/8ca79eae-7193-4c7d-9992-c97902fdce97)

128x144
![horn](https://github.com/sunflower-land/sunflower-land/assets/149222417/2f782a1b-ce42-44b4-9f05-f0b3c0cf50d8)

**Comment by LTNGx39** [comment]
# Bright jellyfish
### **Rarity:** Basic

Sprite:

![Medusa16x](https://github.com/sunflower-land/sunflower-land/assets/131133076/2b1eef7f-f9c0-4413-9b0b-d3d2ebba8218)

Preview:

![Medusa](https://github.com/sunflower-land/sunflower-land/assets/131133076/cd25dce8-ec9d-4764-8095-070bab0149cd)

> **Farm ID:** 161769
> **Discord ID:** ncnlx39

**Comment by IceMonky29** [comment]
Name Fish: Fairy Basslet
Rarity: Basic
Sprite:16x16
![Fairy Basslet](https://github.com/sunflower-land/sunflower-land/assets/142521805/e9fe9318-0ba9-43fc-8f49-af53af9285b1)

Preview:
![Fairy Basslet Big](https://github.com/sunflower-land/sunflower-land/assets/142521805/878d1fc3-2a89-40b3-8e74-d9d14416c27b)


Name: Shrimp
Rarity: Expert
Sprite 16x16:
![Shrimp](https://github.com/sunflower-land/sunflower-land/assets/142521805/0fcb7506-f402-47e9-adc4-b8babb14333c)

Preview:
![Shrimp Big](https://github.com/sunflower-land/sunflower-land/assets/142521805/06304f55-2be4-432c-9a36-366ac94d70c5)

Name: Yellow Fish
Rarity: Advanced
Sprite 16x16:
![Yellow Fish](https://github.com/sunflower-land/sunflower-land/assets/142521805/23373f5a-9c7f-48bc-9016-5a031cf82d12)

Preview:
![Yellow Fis Big](https://github.com/sunflower-land/sunflower-land/assets/142521805/55cbf2ec-d7ee-4a35-9d86-8e7720aef931)

Name: Orca
Rarity: Advance
Sprite: 16x16
 ![orca_16_x_16](https://github.com/sunflower-land/sunflower-land/assets/142521805/84e60add-ff71-4920-8e76-06e50af763c0)

Preview:
![orca 400%](https://github.com/sunflower-land/sunflower-land/assets/142521805/1f44bc43-d112-4f5d-adf7-1acb797b9f9d)
![orca_preview](https://github.com/sunflower-land/sunflower-land/assets/142521805/ee8eaf81-d531-4272-abd1-fa2bc22cfd54)



> **Farm ID:** 162510 
>**Discord ID:** icemonky

**Comment by andand0** [comment]
`Farm ID: 165810`
`Discord: Andando`



### Fish

<hr>

| Name| Rarity | Zoom| 16x16|
|--------|--------|--------|--------|
|Parrot fish|Advance| ![Parrot fish zoom](https://github.com/sunflower-land/sunflower-land/assets/142132586/c09415d4-b127-4925-8aef-5ad81b02d5dc) | ![Parrot fish](https://github.com/sunflower-land/sunflower-land/assets/142132586/b51e0b8b-0b4f-4652-8c19-2d516bdf8617)|
|Golden Pompano| Advance| ![Golden pompano zoom](https://github.com/sunflower-land/sunflower-land/assets/142132586/7d82c353-f3a1-47b7-a95f-1e2a2e3cb31c) |![Golden pompano](https://github.com/sunflower-land/sunflower-land/assets/142132586/80e269e0-0633-4842-8cde-678db278b861)|
|Dorado|Expert| ![Dorado zoom](https://github.com/sunflower-land/sunflower-land/assets/142132586/28ab02cb-50dd-4151-9b0d-3408bf22dd06) | ![Dorado](https://github.com/sunflower-land/sunflower-land/assets/142132586/fba6a9d2-314a-47ec-81e0-2f32b3619bec) |
|Red Grouper| Basic| ![red grouper zoom](https://github.com/sunflower-land/sunflower-land/assets/142132586/c07b5f73-978f-4b2e-b9f7-23d01a05dc95) |![red grouper](https://github.com/sunflower-land/sunflower-land/assets/142132586/3c5f3d28-2e49-43b7-b548-f2d0a031a030)|
|Barracuda| Basic| ![Barracuda zoom](https://github.com/sunflower-land/sunflower-land/assets/142132586/56ff6cec-41ad-46ea-a2b0-25dd5c06ef9a) |![Barracuda](https://github.com/sunflower-land/sunflower-land/assets/142132586/93cda93c-ae14-4fdc-8ea9-a2d5e4d77e4b)|
|Pink Cusk Eel| Expert|  ![Pink Cusk eel zoom](https://github.com/sunflower-land/sunflower-land/assets/142132586/b9f4371c-00e3-4653-b31d-a87c4153bce6) | ![Pink Cusk eel](https://github.com/sunflower-land/sunflower-land/assets/142132586/61e3beea-4b8d-4b97-9922-7b9fb3699a21)|
|Monkfish| Expert| ![Monkfish zoom](https://github.com/sunflower-land/sunflower-land/assets/142132586/7d2d6e4f-1291-46f0-bcbd-873ce3c986bf) |  ![Monkfish](https://github.com/sunflower-land/sunflower-land/assets/142132586/79b54f1c-5162-4e30-9aef-2e214482191b)
|Lobster|Basic|![Lobster zoom](https://github.com/sunflower-land/sunflower-land/assets/142132586/0f1844ff-5e5b-4d87-8481-718703836a9d) | ![Lobster](https://github.com/sunflower-land/sunflower-land/assets/142132586/6afe0539-17fe-4b7f-8379-dfb5e3c9cac2)|
|Queen Triggerfish| Expert |  ![Queen Triggerfish zoom](https://github.com/sunflower-land/sunflower-land/assets/142132586/16ea7754-1d28-4fa3-ae06-38511ec208a7) | ![Queen Triggerfish](https://github.com/sunflower-land/sunflower-land/assets/142132586/b6400d7d-4bf8-4500-aa79-4a31cc76117c) |
|Spotted Trunkfish| Basic|  ![Spotted Trunkfish zoom](https://github.com/sunflower-land/sunflower-land/assets/142132586/833754e5-b934-4adc-95e6-34b95a7b1ba4) | ![Spotted Trunkfish](https://github.com/sunflower-land/sunflower-land/assets/142132586/e954c5e6-20dd-4fcd-96f2-086501ee78fe)|


_I created a fictional fish based on this season._

| Name| Rarity | Zoom| 16x16|
|--------|--------|--------|--------|
| Banana Fish| Expert |  ![Banana Fish zoom](https://github.com/sunflower-land/sunflower-land/assets/142132586/0e8836ec-053b-4a20-957d-c120b12cdc8a) | ![Banana Fish](https://github.com/sunflower-land/sunflower-land/assets/142132586/c10e3a49-f4fb-4413-8f74-88da5cc48959) |

**Comment by peperol** [comment]
The Blobfish , defined as the ugliest in the world..but not for 
me, i think it’s cute:)(unfortunately it is in danger of extinction)

rarity:rare

![IMG_0390](https://github.com/sunflower-land/sunflower-land/assets/114354077/b5f7858c-3e43-4187-9069-505dafce3ad8)

![IMG_0391](https://github.com/sunflower-land/sunflower-land/assets/114354077/0e3254ec-aeb4-455c-97a7-e6cac10bfb9d)

![IMG_6835](https://github.com/sunflower-land/sunflower-land/assets/114354077/3ba94618-be87-4971-a9a6-10b601422efd)



Farm ID: 145849
Discord peperol

**Comment by JcEii** [comment]
**Farm ID:** 15747
**Discord:** jceii


### **First batch submission:**

|Name|Rarity|16x16|Zoom|
|---|---|---|---|
|Spot|Basic|![common_minified1](https://github.com/sunflower-land/sunflower-land/assets/105348036/8811d2c5-9d82-4767-9722-c876daaf8f91)|![common1](https://github.com/sunflower-land/sunflower-land/assets/105348036/287d7c3f-b6f6-488c-810d-ff0162c1ac5b)|
|Weever|Basic|![common_minified2](https://github.com/sunflower-land/sunflower-land/assets/105348036/e483eb18-f81a-41c9-928c-e829cadbc8ac)|![common2](https://github.com/sunflower-land/sunflower-land/assets/105348036/cac9e948-b05d-4d15-91a0-e77dc74da826)|
|Scad fish|Basic|![common_minified3](https://github.com/sunflower-land/sunflower-land/assets/105348036/ff94b1ed-d1a4-4542-b407-86712d0118ff)|![common3](https://github.com/sunflower-land/sunflower-land/assets/105348036/e0051764-cec4-4cb6-bdde-1bf4ef806bbf)|
|Labahita|Basic|![common_minified4](https://github.com/sunflower-land/sunflower-land/assets/105348036/3216f2d1-e750-4a6d-8cb3-ce292a596511)|![common4](https://github.com/sunflower-land/sunflower-land/assets/105348036/f50a065e-cfd1-45e7-b291-1103a75eadc1)|
|Sardine|Basic|![common_minified5](https://github.com/sunflower-land/sunflower-land/assets/105348036/20d38f7c-9e16-4706-b595-0a00c0d9bad2)|![common5](https://github.com/sunflower-land/sunflower-land/assets/105348036/6c3405ea-028d-49a4-94c3-40625333331a)|
|Pig fish|Basic|![common_minified6](https://github.com/sunflower-land/sunflower-land/assets/105348036/533023e2-de26-42b2-affc-7bf63519ce14)|![common6](https://github.com/sunflower-land/sunflower-land/assets/105348036/c9e58d1c-1b10-45f8-a173-bfb465ed3b70)|
|Bermuda Chub|Advance|![advance_minified1](https://github.com/sunflower-land/sunflower-land/assets/105348036/bf8cf826-e7f6-40b2-b90f-f0a2e03cacd7)|![advance1](https://github.com/sunflower-land/sunflower-land/assets/105348036/b8e10f4f-4c5d-4700-af73-072f7aa74eed)|
|Lapu-lapu|Advance|![advance_minified2](https://github.com/sunflower-land/sunflower-land/assets/105348036/717b19cc-d4d9-4baf-8343-fe8f9ec480d8)|![advance2](https://github.com/sunflower-land/sunflower-land/assets/105348036/2701b17d-f367-4644-a0d7-00315d46357b)|
|Jaguar fish|Advance|![advance_minified3](https://github.com/sunflower-land/sunflower-land/assets/105348036/05e6b75b-68d0-4569-9365-a293c6947fb7)|![advance3](https://github.com/sunflower-land/sunflower-land/assets/105348036/b2e5872b-3c3c-4839-b4ba-532f3a3c1af1)|
|Angel fish|Advance|![advance_minified4](https://github.com/sunflower-land/sunflower-land/assets/105348036/bd9ade15-29a4-468e-87d0-62fcefb3e5e5)|![advance4](https://github.com/sunflower-land/sunflower-land/assets/105348036/95e03c5e-daec-4055-9ab8-5108cf8787e3)|
|Halibut|Advance|![advance_minified5](https://github.com/sunflower-land/sunflower-land/assets/105348036/b8c8fe0b-15a4-4718-8319-32058c0988a8)|![advance5](https://github.com/sunflower-land/sunflower-land/assets/105348036/09aae58b-991a-4e0d-9acf-844d88b5cd18)|
|Bangus|Advance|![advance_minified6](https://github.com/sunflower-land/sunflower-land/assets/105348036/13cc33c7-cf8c-4de6-9fe8-f1833425991f)|![advance6](https://github.com/sunflower-land/sunflower-land/assets/105348036/99a3203e-b371-4501-925a-deb676b8a7bd)|
|Tribolodon Carp|Expert|![expert_minified1](https://github.com/sunflower-land/sunflower-land/assets/105348036/78461db4-5ae6-4c4d-9247-5bd6a029c560)|![expert1](https://github.com/sunflower-land/sunflower-land/assets/105348036/a0c6fc6f-1e7d-4cfc-b021-0eeb59602baf)|
|Barracuda|Expert|![barracuda](https://github.com/sunflower-land/sunflower-land/assets/105348036/b30c587f-dbea-4d5d-b48e-0b4b4c3c3ecf)|![barracuda-2](https://github.com/sunflower-land/sunflower-land/assets/105348036/d978c937-51da-4923-a90a-b0fbe141ac4d)|
|Killer Whale|Expert|![expert_minified3](https://github.com/sunflower-land/sunflower-land/assets/105348036/0a70e2a4-d4e4-4acc-a052-a9d0b0b3b484)|![expert3](https://github.com/sunflower-land/sunflower-land/assets/105348036/96b39338-bf1b-4fb4-b3f9-b401e34952ee)|
|Spade fish|Expert|![expert_minified4](https://github.com/sunflower-land/sunflower-land/assets/105348036/60e9dc61-4796-4ffd-9858-554336f59654)|![expert4](https://github.com/sunflower-land/sunflower-land/assets/105348036/6de5e20e-61aa-4f12-a1e6-f848b2b135ce)|
|Croaker|Expert|![expert_minified5](https://github.com/sunflower-land/sunflower-land/assets/105348036/69fab891-a11d-40f5-8b3f-abb7dde06490)|![expert5](https://github.com/sunflower-land/sunflower-land/assets/105348036/47f49885-b60a-44f9-9cc9-a85db837a01c)|
|Goblin fish|Marine marvel|![marine_marvel_minified1](https://github.com/sunflower-land/sunflower-land/assets/105348036/f7a56795-752c-4502-ad1b-9c7a8d07f0a7)|![marine_marvel1](https://github.com/sunflower-land/sunflower-land/assets/105348036/54f1ffb5-c434-4656-92cf-d6da6627b2eb)|

**Comment by SaqibUX** [comment]
**Farm:** 170828
**Discord:** sar_01


| Name| Rarity | 16x16| Zoom|
|--------|--------|--------|--------|
| Blue | Basic | ![blue](https://github.com/sunflower-land/sunflower-land/assets/149222417/6e082096-dda8-4421-ba10-eb6b4f023b31)   | ![Blue64](https://github.com/sunflower-land/sunflower-land/assets/149222417/8837e65c-36fd-4884-9972-6cd011cc9629) |
| Red Striped | Basic | ![Red Sriped 16](https://github.com/sunflower-land/sunflower-land/assets/149222417/0048dd00-4d21-41a3-8327-b067feb8a153)   |   ![Red Sriped 64](https://github.com/sunflower-land/sunflower-land/assets/149222417/c88dddac-954a-4ebb-b024-7b2b520c7f04) |
| Piranha | Advanced |  ![piranha16](https://github.com/sunflower-land/sunflower-land/assets/149222417/85ca103d-e7fe-4141-81d7-7f0dda5803f8)  |  ![piranha64](https://github.com/sunflower-land/sunflower-land/assets/149222417/af08fda5-305f-4bcf-b1b7-34102392510a)  |
| Turtle | Advanced | ![Turtle16](https://github.com/sunflower-land/sunflower-land/assets/149222417/30145773-5d3c-413c-84aa-7be2070db0fb)   | ![Turtle](https://github.com/sunflower-land/sunflower-land/assets/149222417/84ea82ae-65d2-4328-9d9b-5c360904f03a)  |
| Axolotl | Advanced |  ![axolotl16](https://github.com/sunflower-land/sunflower-land/assets/149222417/31bc3ebc-eef5-4e4f-bfc9-587c6158b3d8)  | ![axolotl64](https://github.com/sunflower-land/sunflower-land/assets/149222417/80c4f6f9-f1ae-4798-96db-9f6a29182d6f)   |
| Great white Shark | Expert |  ![white-shark](https://github.com/sunflower-land/sunflower-land/assets/149222417/8618cf43-0b12-4752-b767-7610690f9137)  |  ![white-shark64](https://github.com/sunflower-land/sunflower-land/assets/149222417/45b1599d-91a4-4a9f-8c9d-63ac3c2bcc7c)  |

**Comment by JcEii** [comment]
**Farm ID:** 15747
**Discord:** jceii


### **Second batch submission:**

|Name|Rarity|16x16|Zoom|
|---|---|---|---|
|Shrimp|Basic|![basic1](https://github.com/sunflower-land/sunflower-land/assets/105348036/947a2375-5791-4868-bb57-9ce8ea73c219)|![basic_big1](https://github.com/sunflower-land/sunflower-land/assets/105348036/388f7b49-f703-40dc-b82e-ca929a62dac3)|
|Goatfish|Basic|![basic2](https://github.com/sunflower-land/sunflower-land/assets/105348036/ff3d4e33-5aac-4a94-b874-364138f6f720)|![basic_big2](https://github.com/sunflower-land/sunflower-land/assets/105348036/7f1a6942-a5d5-4cbc-9058-4210044a98a7)|
|Herring|Basic|![basic3](https://github.com/sunflower-land/sunflower-land/assets/105348036/6e3c8278-ab0c-45d7-b8dd-78c7bb55ad94)|![basic_big3](https://github.com/sunflower-land/sunflower-land/assets/105348036/2dc9e432-a7c7-46e7-bc9e-ba10045361b3)|
|Stonefish|Advance|![advance1](https://github.com/sunflower-land/sunflower-land/assets/105348036/6a42632d-2009-4ddc-8056-80a999718405)|![advance_b1](https://github.com/sunflower-land/sunflower-land/assets/105348036/2df34407-e75a-4c82-a035-c0697984cc0e)|
|Albacore|Advance|![advance2](https://github.com/sunflower-land/sunflower-land/assets/105348036/025a9748-c440-444e-a6bc-cdfe3145f056)|![advance_b2](https://github.com/sunflower-land/sunflower-land/assets/105348036/5c33b2a6-5d45-4df2-bc4f-898f80d62f1d)|
|Tulingan|Advance|![advance3](https://github.com/sunflower-land/sunflower-land/assets/105348036/e38624c8-1267-4a8a-a289-ac0262528436)|![advance_b3](https://github.com/sunflower-land/sunflower-land/assets/105348036/011f199f-ca7c-4b74-a9b9-d30570182d4b)|
|Cobia|Advance|![advance4](https://github.com/sunflower-land/sunflower-land/assets/105348036/21957b92-18a0-499f-8560-fcbe158807c1)|![advance_b4](https://github.com/sunflower-land/sunflower-land/assets/105348036/8489f70b-ef8b-4b08-80c8-45dada003b7c)|
|Jack|Expert|![expert1](https://github.com/sunflower-land/sunflower-land/assets/105348036/ff300155-5c05-44e7-8643-a487e0f83280)|![expert_b1](https://github.com/sunflower-land/sunflower-land/assets/105348036/0ed84af7-4048-4dc0-b85f-d36b706d33ad)|
|Red Mullet|Expert|![expert2](https://github.com/sunflower-land/sunflower-land/assets/105348036/79872470-9515-40ea-a719-13195e95bac5)|![expert_b2](https://github.com/sunflower-land/sunflower-land/assets/105348036/4e5f3749-b0c1-4da5-81a4-a122af19675f)|
|Frilled Shark|Expert|![expert4](https://github.com/sunflower-land/sunflower-land/assets/105348036/22ff4689-cb13-4dbd-b7b5-cf501b3f10c9)|![expert_b4](https://github.com/sunflower-land/sunflower-land/assets/105348036/fac38506-561c-4725-aa58-493627c97b7a)|
|leedsichthys|Expert|![expert5](https://github.com/sunflower-land/sunflower-land/assets/105348036/bf8750e6-6b5b-4b9a-b90b-8d7962c8e0db)|![expert_b5](https://github.com/sunflower-land/sunflower-land/assets/105348036/88f60f22-cb09-4bf5-b681-997152ad1cf0)|
|helicoprion|Expert|![expert6](https://github.com/sunflower-land/sunflower-land/assets/105348036/0241adee-417a-45d2-9c46-d6c72be859f5)|![expert_b6](https://github.com/sunflower-land/sunflower-land/assets/105348036/31037bae-44da-4c79-bd40-1cf2de8e964c)|
